assets/js/components/icons/star.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+import { Icon } from '@wordpress/components';
+
+export default () => (
+	<Icon
+		icon={
+			<svg
+				xmlns="http://www.w3.org/2000/svg"
+				width="18"
```

Comment:
I noticed all the other icons were set to 24px - any reason for this one being 18?

Reply:
No idea, but that's what I get from [the material icons website.](https://material.io/tools/icons/) I think it's fine, it'll scale up to fit the dimensions needed.